src/lib/VTXSPI/devVTXSPI.cpp

```diff
@@ -83,7 +84,21 @@ static void rtc6705WriteRegister(uint32_t regData)
         digitalWrite(GPIO_PIN_SPI_VTX_NSS, LOW);
     }
 
-    #if defined(PLATFORM_ESP32)
+    #if defined(PLATFORM_ESP32_S3)
+        // On the S3 there's a hardware "limitation", sometimes when sending 25 bits the last
+        // bit is 1, when it should be 0. See https://github.com/espressif/esp-idf/issues/8487#issuecomment-1059641006
+        //
+        // To reproduce, change from channel A4 to A1, then A1 to A4 (ok), then A4 to A3, then A3 to A4 (fail)
+        // 12816, 9286833 appears on the scope when changing from A:1->A:4
+        // 12816, 26064049 appears on the scope when changing from A:3->A:4
+        //
+        // 9286833  = 0_1000_1101_1011_0100_1011_0001
+        // 26064049 = 1_1000_1101_1011_0100_1011_0001
+        //
+        // so, to workaround this issue, we send 32 bits, and the RTC67005 ignores the last 7 bits.
+        regData |= 0b1111111 << 25;
+        vtxSPI->transferBits(regData, nullptr, 32);
```

Comment:
Let just do this for both S3 and esp32. There is no harm and itll remove some defines.

Reply:
I would prefer to keep the defines and related comments, intact, until the root cause has been established.

Reply:
ahh, as @pkendall64 found in https://github.com/ExpressLRS/ExpressLRS/pull/2406#issuecomment-1722573356 it's a silicon bug.

@pkendall64 Can you remove the VTX workaround commit from this PR, and I'll update my other PR which can then be merged before or after this one.

Reply:
I removed this commit from this PR and left it for #2406